posthog/queries/test/test_lifecycle.py

```diff
@@ -881,26 +890,18 @@ def test_timezones(self):
         self.team.save()
 
         result_pacific = Trends().run(
-            Filter(
-                data={
-                    "date_from": "2020-01-12T00:00:00Z",
-                    "date_to": "2020-01-19T00:00:00Z",
-                    "events": [{"id": "$pageview", "type": "events", "order": 0}],
-                    "shown_as": TRENDS_LIFECYCLE,
-                },
-                team=self.team,
-            ),
+            Filter(data=filter_data),
             self.team,
         )
-        assertLifecycleResults(
+        self.assertLifecycleResults(
             result_pacific,
             [
                 {
                     "status": "dormant",
-                    "data": [-1.0, -2.0, -1.0, 0.0, -2.0, 0.0, -1.0, 0.0],
+                    "data": [0.0, -1.0, -2.0, -1.0, 0.0, -2.0, 0.0, -1.0],
                 },
-                {"status": "new", "data": [1, 0, 0, 1, 0, 0, 0, 0]},
-                {"status": "resurrecting", "data": [1, 1, 0, 1, 0, 1, 0, 1]},
+                {"status": "new", "data": [1, 1, 0, 0, 1, 0, 0, 0]},
+                {"status": "resurrecting", "data": [0, 1, 1, 0, 1, 0, 1, 0]},
                 {"status": "returning", "data": [0, 0, 0, 0, 0, 0, 0, 0]},
```

Comment:
How come the query results changed in an old non-hogql insight?

Ideally we want to completely `rm -rf` this entire `posthog/queries` folder, or at least the insights contained within. Modifying those existing tests seems like a bit of wasted effort... or I'm likely just not seeing it 🤔